tests/test_backup_and_restore.py

```diff
@@ -0,0 +1,48 @@
+from unittest.mock import MagicMock, call
+
+from bin.bb8.restore import run_restore
+from bin.bb8.backup import run_backup
+from tests.mocks import mock_target, mock_settings, Dynamic
+
+
+def test_backup_is_only_called_for_targets_with_backup_set_to_true():
+    # Setup
+    targets = [
+        mock_target("a", backup=True),
+        mock_target("b", backup=False),
+        mock_target("c", backup=True)
+    ]
+    settings = mock_settings(targets)
+    rsync = Dynamic("rsync", backup_volume=MagicMock())
+
+    # Test
+    run_backup(lambda: settings, rsync)
+
+    # Check
+    rsync.backup_volume.assert_has_calls([
+        call(settings, "mount-a"),
```

Comment:
I'm totally unfamiliar with this is framework - is this checking that only these two calls happen (i.e., that "b" is not backed up) or that _at least_ these two calls happen?

Reply:
ok, just confirmed that this is the actual list of calls